formatting/key_pair_id_test.go

```diff
@@ -0,0 +1,21 @@
+package formatting
+
+import "testing"
+
+type KeypairIDTest struct {
+	input  string
+	output string
+}
+
+var keypairIDTests = []KeypairIDTest{
```

Comment:
We have https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md. You have no naming issues when you for instance put the structure inline.

Reply:
Done